app/javascript/data/colors.json

```diff
@@ -28,7 +28,7 @@
   },
   "climate": {
     "ramp": [ "#EDD093", "#BFBC35", "#007F53", "#123A33"],
-    "density": "#895122",
+    "carbon": ["#895122", "#898522", "#892227"],
```

Comment:
These colours (specially #895122 and #892227) look a bit too similar with low contrast or night shift (1.4 contrast ratio)

Reply:
We chose these from the initial hex code for carbon based on the chroma scale. Waiting for their feedback on these too.